gatsby-theme-landing-page/README.md

```diff
@@ -38,9 +38,25 @@ CONTENTFUL_SPACE_ID="<SPACE_ID>"
 CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
 ```
 
-<!-- TODO: update these instructions once the JSON file is created -->
+Import the content model and demo data to your Contentful space
 
-Upload the `data/sample-data.json` to your Contentful space to set up the content model. Then, create your first landing page in Contentful.
+1. Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
```

Comment:
If this is a single npm command, I'd say we should inline the instructions in a code block here, e.g. `npm i -g contentful-cli`

Reply:
yeah, there's even `npx contentful-cli <cmd>`